imageroot/actions/restore-module/40restore-databases

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+set -e -o pipefail
+exec 1>&2 # Redirect any output to the journal (stderr)
+
+# nothing to restore we exit
+if [ ! -d restore/ ]; then
+    exit 0
+fi
+
```

Comment:
If the core does not restore the subdir something is broken... I prefer to not ignore that condition and let Bash do its error handling on line 20.

```suggestion
```